portfolio/src/main/webapp/script.js

```diff
@@ -12,22 +12,72 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
+// /**
+//  * Adds a random greeting to the page.
+//  */
 
+// function addRandomGreeting() {
+//   const greetings =
+//       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
+//   // Pick a random greeting.
+//   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
+//   // Add it to the page.
+//   const greetingContainer = document.getElementById('greeting-container');
+//   greetingContainer.innerText = greeting;
+// }
+
+window.onload = function () {
+    // document.getElementsById('all').checked=true;
+    showAllColumns('all') 
+}
 
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function showAllColumns(column) {
+  var arrayOfAllColumns, i;
+  arrayOfAllColumns = document.getElementsByClassName("column");
```

Comment:
If you have some free time, look up 'hungarian notation' and what that is. Basically it's the idea of including type information (like 'array') in a variable name. It's fallen out of favor, and is generally avoided in strongly typed languages. Since JS is a weakly typed language, I have no problems with it here.

Reply:
Do you mean its okay if I use Hungarian Notation in Js? Can you expatiate on what you mean by weakly typed language?

Reply:
sorry, the hungarian notation was simply to help give you some background on the idea of encoded type information into variable names. hungarian notation has mostly fallen out of favor these days - i would only use it if it's part of the style for an existing codebase. this is fine as is.